parsers/to-css-text-style/to-css-text-style.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('to-css-text-style', () => {
         '.ts-with-allan-again {\n' +
           '  color: rgb(196, 196, 196);\n' +
           '  font-family: Allan;\n' +
-          '  font-size: 12px;\n' +
+          '  font-size: 12.1px;\n' +
```

Comment:
12.1px is really weird. Is this normal?? Why would we promote such a weird font-size value.

Reply:
This is because of the seed. The seeds must allow test for every parsers. If we don't use random values like these, there is no way we can validate th tests. The tests are unit tests so we must test only one parser at a time hence the strange values